tests/gis_tests/maps/tests.py

```diff
@@ -4,8 +4,10 @@
 from unittest import skipUnless
 
 from django.contrib.gis.geos import HAS_GEOS
+from django.contrib.gis.maps.google.overlays import GEvent, GOverlayBase
```

Comment:
Looks like this import triggers a test failure if you don't gave the GEOS dependency installed. It should be imported in the actual test.